src/routes/postTransition.ts

```diff
@@ -141,6 +143,27 @@ const postTransitionEndpoints = (router: Router, dependencies: IDependencies): v
     companyAuthentication,
     dependencies.limitedPartnerPostTransitionController.sendPageData()
   );
+
+  router.get(
+    ADD_LIMITED_PARTNER_LEGAL_ENTITY_URL,
+    companyAuthentication,
+    dependencies.limitedPartnerPostTransitionController.getPageRouting()
+  );
+  router.post(
+    ADD_LIMITED_PARTNER_LEGAL_ENTITY_URL,
+    companyAuthentication,
+    dependencies.limitedPartnerPostTransitionController.createLimitedPartner()
+  );
```

Comment:
should have empty line between the different url router get/post groups

Reply:
Added the line